modules/authorization/resource-scope-role-assignment/README.md

```diff
@@ -63,7 +63,7 @@ module storageAccount 'br/public:storage/storage-account:1.0.1' = {
 }
 
 // module call with all parameters
-module roleAssignmentOperator '../main.bicep' = {
+module roleAssignmentOperator 'br/public:authorization/resource-scope-role-assignment:1.0.0' = {
```

Comment:
```suggestion
module roleAssignmentOperator 'br/public:authorization/resource-scope-role-assignment:1.0.1' = {
```

It might be a bit counterintuitive, but the module version always starts with `*.*.1` which is set by https://github.com/dotnet/Nerdbank.GitVersioning.